PocketKit/Sources/PocketKit/Home/SlateDetailViewModel.swift

```diff
@@ -33,14 +33,16 @@ class SlateDetailViewModel {
     private let user: User
     private let userDefaults: UserDefaults
     private var subscriptions: [AnyCancellable] = []
+    private let featureFlags: FeatureFlagService
 
-    init(slate: Slate, source: Source, tracker: Tracker, user: User, userDefaults: UserDefaults) {
+    init(slate: Slate, source: Source, tracker: Tracker, user: User, userDefaults: UserDefaults, featureFlags: FeatureFlagService) {
```

Comment:
should we also add these to the tests inits? otherwise, looks good!

Reply:
Yeah the tests are crashing right meow, fixing!